### ISSUE TITLE
rmagick 4.2.4 not installing in Ubuntu 20

### ISSUE THREAD

**Issue by robertochingon** [rmagick/rmagick#1327]
I´m installing a fresh copy of a ruby on rails web app in a new Amazon EC2 micro instance running `Ubuntu 20.04.3 LTS (GNU/Linux 5.11.0-1022-aws x86_64)`.

My ruby version is: `ruby 3.0.3p157 (2021-11-24 revision 3fb7d2cadc) [x86_64-linux]`

and if I run `convert -version`, my imagemagick version is: Version: `Version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org`

When I run bundle install at first time, I got an error notifying that I was missing some libraries. I installed some developer libraries according to other posts. 

When I run bundle update now, when it gets to rmagick gem, I don´t see any error anymore, but it get stuck. It can be there for hours and nothing happens.

Fetching rmagick 4.2.4

Installing rmagick 4.2.4 with native extensions
...

I don´t know what is happening and what to check.

**UPDATE**

For debugging purposes, I have created a new instance and restarted the installation process, maybe it´s easier this way. After running bundle install before installing any rmagick libraries I get next error:

    Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
    
        current directory: /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick
    /home/ubuntu/.rbenv/versions/3.0.3/bin/ruby -I /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0 -r ./siteconf20220328-26859-jyvo9y.rb extconf.rb
    checking for brew... no
    checking for pacman... no
    checking for Ruby version >= 2.3.0... yes
    checking for pkg-config... yes
    Package MagickCore was not found in the pkg-config search path.
    Perhaps you should add the directory containing `MagickCore.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'MagickCore' found
    
    
    ERROR: Can't install RMagick 4.2.4.
    Can't find the ImageMagick library or one of the dependent libraries.
    Check the mkmf.log file for more detailed information.
    
    
    *** extconf.rb failed ***
    Could not create Makefile due to some reason, probably lack of necessary
    libraries and/or headers.  Check the mkmf.log file for more details.  You may
    need configuration options.
    
    Provided configuration options:
    	--with-opt-dir
    	--without-opt-dir
    	--with-opt-include
    	--without-opt-include=${opt-dir}/include
    	--with-opt-lib
    	--without-opt-lib=${opt-dir}/lib
    	--with-make-prog
    	--without-make-prog
    	--srcdir=.
    	--curdir
    	--ruby=/home/ubuntu/.rbenv/versions/3.0.3/bin/$(RUBY_BASE_NAME)
    
    To see why this extension failed to compile, please check the mkmf.log which can be found here:
    
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/mkmf.log
    
    extconf failed, exit code 1
    
    Gem files will remain installed in /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4 for inspection.
    Results logged to /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/gem_make.out
    
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:93:in `run'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/ext_conf_builder.rb:47:in `block in build'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/tempfile.rb:317:in `open'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/ext_conf_builder.rb:26:in `build'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:159:in `build_extension'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:193:in `block in build_extensions'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:190:in `each'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:190:in `build_extensions'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/installer.rb:845:in `build_extensions'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/rubygems_gem_installer.rb:71:in `build_extensions'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/rubygems_gem_installer.rb:28:in `install'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/source/rubygems.rb:200:in `install'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/gem_installer.rb:54:in `install'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/gem_installer.rb:16:in `install_from_spec'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/parallel_installer.rb:186:in `do_install'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/parallel_installer.rb:177:in `block in worker_pool'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:62:in `apply_func'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:57:in `block in process_queue'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:54:in `loop'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:54:in `process_queue'
      /home/ubuntu/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:91:in `block (2 levels) in create_threads'
    
    An error occurred while installing rmagick (4.2.4), and Bundler cannot continue.
    
    In Gemfile:
      rmagick

**Comment by Watson1978** [comment]
I'm guessing that you need to run `sudo apt-get install libmagickwand-dev` before.

Ref. https://github.com/rmagick/rmagick#ubuntu

**Comment by Watson1978** [comment]
If you have not been solved your issue yet, feel free to reopen this ticket.

**Comment by SahSantoshh** [comment]
Facing same issue with Ubuntu 21.1

```shell
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick
/home/santosh/.rbenv/versions/3.0.3/bin/ruby -I /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0 -r ./siteconf20220405-66157-ga7ufv.rb
extconf.rb
checking for brew... yes
checking for Ruby version >= 2.3.0... yes
checking for pkg-config... yes


ERROR: Can't install RMagick 4.2.4. Can't find ImageMagick with pkg-config


*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
	--with-opt-dir
	--without-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/home/santosh/.rbenv/versions/3.0.3/bin/$(RUBY_BASE_NAME)

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4 for inspection.
Results logged to /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/gem_make.out

  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:93:in `run'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/ext_conf_builder.rb:47:in `block in build'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/tempfile.rb:317:in `open'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/ext_conf_builder.rb:26:in `build'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:159:in `build_extension'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:193:in `block in build_extensions'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:190:in `each'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/ext/builder.rb:190:in `build_extensions'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/rubygems/installer.rb:845:in `build_extensions'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/rubygems_gem_installer.rb:71:in `build_extensions'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/rubygems_gem_installer.rb:28:in `install'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/source/rubygems.rb:200:in `install'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/gem_installer.rb:54:in `install'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/gem_installer.rb:16:in `install_from_spec'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/parallel_installer.rb:186:in `do_install'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/installer/parallel_installer.rb:177:in `block in worker_pool'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:62:in `apply_func'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:57:in `block in process_queue'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:54:in `loop'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:54:in `process_queue'
  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0/bundler/worker.rb:91:in `block (2 levels) in create_threads'

An error occurred while installing rmagick (4.2.4), and Bundler cannot continue.

In Gemfile:
  rmagick
```
PATHS in my `.zshrc` file
```shell
export PATH="$PATH":"/usr/lib/x86_64-linux-gnu/ImageMagick-6.9.11/bin-q16"
export PKG_CONFIG_PATH="/usr/lib/x86_64-linux-gnu/pkgconfig"
```

LOGS
`cat   /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/mkmf.log`
```shell
find_executable: checking for brew... -------------------- yes
assert_minimum_ruby_version!: checking for Ruby version >= 2.3.0... -------------------- yes
find_executable: checking for pkg-config... -------------------- yes

ERROR: Can't install RMagick 4.2.4. Can't find ImageMagick with pkg-config
```
ImageMagick in pkgConfig Directory
`ls /usr/lib/x86_64-linux-gnu/pkgconfig/ | grep Image`

```shell
ImageMagick-6.Q16.pc
ImageMagick-im6.pc
ImageMagick-im6.q16.pc
ImageMagick.pc
```

**Comment by Watson1978** [comment]
Please attach `/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/mkmf.log` or log of `gem install rmagick --verbose`, (or both)

**Comment by Watson1978** [comment]
I'm guessing that your issue will be solve with https://github.com/rmagick/rmagick/pull/1325

**Comment by SahSantoshh** [comment]
Logs of `gem install rmagick --verbose`
```
HEAD https://index.rubygems.org/
200 OK
GET https://index.rubygems.org/info/rmagick
200 OK
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.codeclimate.yml
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.editorconfig
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.github/ISSUE_TEMPLATE.md
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.github/workflows/ci.yml
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.gitignore
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.rspec
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.rubocop.yml
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.rubocop_todo.yml
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/.yardopts
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/CHANGELOG.md
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/CODE_OF_CONDUCT.md
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/CONTRIBUTING.md
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/Gemfile
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/LICENSE
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/README.md
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/Rakefile
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/before_install_linux.sh
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/before_install_osx.sh
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/deprecated/RMagick.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/extconf.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmagick.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmagick.h
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmdraw.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmenum.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmfill.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmilist.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmimage.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rminfo.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmkinfo.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmmain.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmmontage.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmpixel.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmstruct.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick/rmutil.c
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rmagick.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rmagick/version.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rmagick_internal.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/clippath.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/container.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/deep_equal.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/describable.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/embellishable.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/misc.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/paint.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/pathdata.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/rvg.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/stretchable.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/stylable.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/text.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/transformable.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/lib/rvg/units.rb
/home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/rmagick.gemspec
Building native extensions. This could take a while...
current directory: /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick
["/home/santosh/.rbenv/versions/3.0.3/bin/ruby", "-I", "/home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0", "-r", "./siteconf20220405-49631-ysdt9a.rb", "extconf.rb"]
checking for brew... yes
checking for Ruby version >= 2.3.0... yes
checking for pkg-config... yes


ERROR: Can't install RMagick 4.2.4. Can't find ImageMagick with pkg-config


*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
	--with-opt-dir
	--without-opt-dir
	--with-opt-include
	--without-opt-include=${opt-dir}/include
	--with-opt-lib
	--without-opt-lib=${opt-dir}/lib
	--with-make-prog
	--without-make-prog
	--srcdir=.
	--curdir
	--ruby=/home/santosh/.rbenv/versions/3.0.3/bin/$(RUBY_BASE_NAME)
ERROR:  Error installing rmagick:
	ERROR: Failed to build gem native extension.

    current directory: /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4/ext/RMagick
/home/santosh/.rbenv/versions/3.0.3/bin/ruby -I /home/santosh/.rbenv/versions/3.0.3/lib/ruby/3.0.0 -r ./siteconf20220405-49631-ysdt9a.rb extconf.rb
Building has failed. See above output for more information on the failure.
To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/gems/rmagick-4.2.4 for inspection.
Results logged to /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/gem_make.out
```
mkmg.logs
```
cat /home/santosh/.rbenv/versions/3.0.3/lib/ruby/gems/3.0.0/extensions/x86_64-linux/3.0.0/rmagick-4.2.4/mkmf.log

find_executable: checking for brew... -------------------- yes

--------------------

assert_minimum_ruby_version!: checking for Ruby version >= 2.3.0... -------------------- yes

--------------------

find_executable: checking for pkg-config... -------------------- yes

--------------------

ERROR: Can't install RMagick 4.2.4. Can't find ImageMagick with pkg-config
```

**Comment by mockdeep** [comment]
I've published the fix for the OP's issue as version 4.2.5.
